sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java

```diff
@@ -67,7 +67,7 @@ static OtlpGrpcSpanExporter configureOtlpSpans(ConfigProperties config) {
     ClasspathUtil.checkClassExists(
         "io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter",
         "OTLP Trace Exporter",
-        "opentelemetry-exporter-otlp-trace");
+        "opentelemetry-exporter-otlp");
```

Comment:
Why this change?

Reply:
Oops forgot to add to description, updated